### PR TITLE
Add DynamicPolicy scaleNodesExpression documentation

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

File: src/docs/admin/references/NodeSourcePolicyReference.adoc [hunk 1]
```diff
@@ -299,9 +299,27 @@ This policy will never acquire more nodes than the infrastructure is capable of
...
+----
+
+Two additional parameters allow to fine-tune this formula:
+
+ * loadFactorRoundingUp: when `false`, `Math.ceil` is replaced by `Math.floor` in the above formula (rounding down instead of rounding up).
+ * loadFactorLegacyBehavior: when `true`, the formula is transformed to:
+----
+expectedNbNodes = Math.ceil(nbPendingTasks / loadFactor + nbBusyNodes)
+----
+This means that new nodes will be deployed as long as there are pending tasks remaining, regardless of the *loadFactor* value. In that case, *loadFactor* controls only how many new Nodes are deployed per policy cycle and not how many Nodes are present in the Node Source with regard to the number of tasks to run.
```
Comment (mboussaa) [inline src/docs/admin/references/NodeSourcePolicyReference.adoc:319]: nodes => Nodes

**Review by mboussaa — APPROVED** [review]

File: src/docs/admin/references/NodeSourcePolicyReference.adoc (hunk 1 above)
Comment (fviale) [reply src/docs/admin/references/NodeSourcePolicyReference.adoc:319]: indeed

File: src/docs/admin/references/NodeSourcePolicyReference.adoc [hunk 2]
```diff
@@ -299,9 +299,27 @@ This policy will never acquire more nodes than the infrastructure is capable of
...
 
 The DynamicPolicy will scale up when pending tasks are detected in the ProActive Scheduler queue and the Node Source does not have enough free nodes to assign.
+
 In some cases, the existing free nodes will not match the requirements of these pending tasks. This can happen when the task contains a <<../user/ProActiveUserGuide.adoc#_selection_scripts,Selection Script>> or a <<../user/ProActiveUserGuide.adoc#_generic_information_selection,NODE_ACCESS_TOKEN>> restriction that discards existing free nodes.
 In that case, the Dynamic Policy will use a *best-effort strategy* to handle these tasks that remain in pending state. When tasks remain in pending state longer than `initDelay` milliseconds, the node source will trigger a scale-up (one new node per pending task).
 
+The *loadFactor* is the main parameter that allows to control the scale up of the DynamicPolicy. It decides how many new ProActive Nodes will be deployed when considering the current number of pending and running tasks. *loadFactor* is a ratio which computes the expected number of Nodes in the Node Source according to the formula:
```
Comment (aminelouati) [inline src/docs/admin/references/NodeSourcePolicyReference.adoc:306]: to control the scale up --> to control the scale up/down ?

File: src/docs/admin/references/NodeSourcePolicyReference.adoc (hunk 1 above)
Comment (aminelouati) [reply src/docs/admin/references/NodeSourcePolicyReference.adoc:319]: to run -> to be executed

**Review by aminelouati — APPROVED** [review]

File: src/docs/admin/references/NodeSourcePolicyReference.adoc (hunk 1 above)
Comment (fviale) [reply src/docs/admin/references/NodeSourcePolicyReference.adoc:319]: @aminelouati that would be too heavy.

File: src/docs/admin/references/NodeSourcePolicyReference.adoc (hunk 2 above)
Comment (fviale) [reply src/docs/admin/references/NodeSourcePolicyReference.adoc:306]: ok

**Pull request merged**